app/pages/cif/projects.tsx

```diff
@@ -152,8 +138,7 @@ export function Projects({ preloadedQuery }: RelayProps<{}, projectsQuery>) {
   ) : (
     <Button
       role="button"
-      onClick={handleCreateProject}
-      disabled={isCreatingProject}
+      onClick={() => router.push(getNewProjectRevisionPageRoute())}
```

Comment:
We need to add this logic to the new project button on the home page too, right now it's erroring out for me.